recipes/barvinok/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set version = "0.41.5" %}
+
+package:
+  name: barvinok
+  version: {{ version }}
+
+source:
+  url: http://barvinok.gforge.inria.fr/barvinok-{{ version }}.tar.gz
+  sha256: 6834ed7588fcc60f1d96743cf6e4c23c3174a0cdd72c128e1d0d9714957145c0
+
+build:
+  number: 0
+  skip: true  # [win]
```

This will need a `run_exports` specify the ABI compability if things are built against it. This could for example be the following:
```suggestion
  skip: true  # [win]
  run_exports:
    - {{ pin_subpackage(barvinok, max_pin="x.x.x" }}
```